scopes/design/ui/error-page/error-page.compositions.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ThemeCompositions } from '@teambit/documenter.theme.theme-compositions';
+import { ErrorPage } from './error-page';
+
+export const Error404 = () => (
+  <ThemeCompositions style={{ minHeight: 400 }}>
```

Comment:
No need anymore to add a style to the ThemeComposition to override the minHeight of 100vh because I removed it and move it to be a class that can be used if needed, so now the ThemeCompositions not have it by default.
So you can remove all the style override.
I already removed all the unnecessary style in Bit

Reply:
I do the changes in your branch now and I'll update it soon.